src/index.ts

```diff
@@ -26,8 +27,37 @@ client.once('ready', () => {
             username: `${client.user.username}#${client.user.discriminator} (${client.user.id})`,
             prefix: config.prefix,
         },
-        'Client ready',
+        'Bot ready',
     );
+
+    if (config.guildID) {
```

Comment:
How do we want to register commands? This is no longer the "recommended" way to do this. Recommended is with a separate script. but that makes development harder to iterate

Reply:
It also requires setting the client id and not just the token

Reply:
The recommended way is starting to make sense to me. It can take a while for the commands to load.

We can include this automatically in the Docker image (and make this the recommended way to run).

For development, we could add a new script and include that in concurrently.